gwtp-core/gwtp-mvp-client/src/main/java/com/gwtplatform/mvp/client/PresenterWidget.java

```diff
@@ -249,6 +264,18 @@ public void clearSlot(Object slot) {
         clearSlot(LegacySlotConvertor.convert(slot));
     }
 
+    /**
+     * This method clears the content in a specific slot. No
+     * {@link com.gwtplatform.mvp.client.proxy.ResetPresentersEvent} is fired.
+     * The attached {@link View} should manage this slot when its
+     * {@link View#setInSlot(Object, com.google.gwt.user.client.ui.Widget)} is called. It should also clear
+     * the slot when the {@link View#setInSlot(Object, com.google.gwt.user.client.ui.Widget)} method is
+     * called with {@code null} as a parameter.
+     * <p/>
+     * For more details on slots, see {@link HasSlots}.
+     *
+     * @param slot Specific slot to clear.
+     */
```

Comment:
Is this copied from `HasSlot`? The generated javadoc should duplicate the description already.